test/functional/fixtures/regression/gh-2205/testcafe-fixtures/index.js

```diff
@@ -0,0 +1,27 @@
+import { Selector } from 'testcafe';
+
+fixture `GH-2205 - Should type in div if it has an invisible child with contententeditable=false`
+    .page `http://localhost:3000/fixtures/regression/gh-2205/pages/index.html`;
+
+async function typeAndCheck (t, editorId) {
+    const editor = Selector(editorId);
+
+    await t
+        .click(editor)
+        .typeText(editor, 'Hello')
+        .wait(1000)
```

Comment:
Why we need to wait? Can we increase `selectorTimeout` [here](https://github.com/DevExpress/testcafe/pull/2222/files#diff-8df52206dc76dcf280091c93acb7d233R2)?